homeassistant/components/myuplink/sensor.py

```diff
@@ -54,13 +55,27 @@
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UnitOfPressure.BAR,
     ),
+    "days": SensorEntityDescription(
+        key="days",
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTime.DAYS,
+        suggested_display_precision=0,
+    ),
```

Comment:
There are already multi sensors reporting time, are the ones added in this PR the same, just with different units?

Reply:
Same reason as above. This is to keep the default unit the same as in myUplink app for each datapoint. The API can expose many different points of duration data, in some of them seconds is the relevant unit in others days are relevant. I think it will be a better user experience if we stick to the defaults that the API suggests in the metadata.